### ISSUE TITLE
Alternative vol/page numbers are not shown properly in suttaplex nerdy row

### ISSUE THREAD

**Issue by sujato** [suttacentral/suttacentral#1474]
#### Description:

In some cases, the PTS 9and possibly other) have published the same text in two editions, with different vol/age numbers. We have the data for both editions, and it should be displayed.

By default we show the *latest* edition. However reference works such as the PTS dictionary will often have the early edition.

Thus both editions should be shown as a tooltip when hovering. However currently this is broken: the tooltip displays blank data. Here is an example:

```
<span class="vol-page nerdy-row-element" title="PTS volume and page:
1st edition = {pts1}
2nd edition = {pts2}">
              SN i 64
            </span>
```

#### Steps to reproduce:
Go to:

https://suttacentral.net/sn1-adittavagga

Hover over vol/page on suttaplex.

#### Expected behaviour:

Where there are multiple editions, show the data on hover in suttaplex.

**Comment by ihongda** [comment]
![深度截图_选择区域_20190903020406](https://user-images.githubusercontent.com/4037966/64135384-6b872000-ce1a-11e9-85d0-1090b805c407.png)

This bug is fixed.

**Comment by sujato** [comment]
Unfortunately, I think this has been broken again by my recent updates to the uid-expansion. Sorry about that! 

In the past, we used `pts1` and `pts2` to mean "PTS first edition" and "PTS second edition". However this created errors, we also want to say `pts1.66` is "volume 1, page 66". So the first number after `pts` sometimes meant "edition" and sometimes meant "volume".

The current form of UID avoids this by using the form `pts1ed` and `pts2ed`. 

In addition, we now always specify the edition in full, that is it must always be `pts-vp-pli1ed` or `pts-vp-pli-2ed`.

The meaning of them is:

- `pts-vp-pli1ed`: "Volume and page of the first Pali edition of the Pali Text Society."
- `pts-vp-pli-2ed`: "Volume and page of the second Pali edition of the Pali Text Society."

These are found in `uid_expansion`:

https://github.com/suttacentral/sc-data/tree/master/misc

**Comment by ihongda** [comment]
Okey, Bhante, I will adjust according to the latest situation 😄

**Comment by chansikpark** [comment]
I see that this behaviour relies on volpages in the database being stored in "//"-separated format (eg. "SN i 32//SN i 66"). I believe currently, the only source of volpages that take advantage of this are in `sutta.json` and only for the first volume of SN. The other two sources of volpages is from pootle files and legacy html files.

In case we want to have more of the texts showing alternative vol/page numbers prior to bilara-data integration (ie for a small change), I pushed a change [here](https://github.com/chansikpark/suttacentral/commit/e38a5c13f621264e0fd21256616b127ce37b15ed), which depends on PR #1523. It loads multiple volpages into the database for legacy Pali texts in '//'-separated format.

Eg. for Buddhavamsa:
![Screenshot from 2019-10-14 02-01-16](https://user-images.githubusercontent.com/16063972/66719171-21388c00-ee27-11e9-9e20-06b495175622.png)

Tangentially, there's some logic in the frontend [here](https://github.com/suttacentral/suttacentral/blob/master/client/elements/suttaplex/card/sc-suttaplex.js#L143) which I'm guessing is the reason why cases where vol/page numbers between the two PTS editions are identical show up just as "Volume and page" in the tooltip (eg. SN 1.1, SN 1.2). I'm wondering whether this might lead to a bit of uncertainty in some cases.

**Comment by sujato** [comment]
Chan, thanks for this. I'm a little unclear right now exactly what must be done to fix this issue once and for all. Can I ask you to specify the steps needed and then we can ensure it is completed.

**Comment by chansikpark** [comment]
🙏

The topic branch I created in-fork called '1474-tooltip-volpages' resolves this issue. It branches off of and so includes PR #1523 (which addresses #1518). Therefore, I'd just close the old one and refer back to it in a new PR. This new PR would resolve two issues: this one and #1518.

What I'd like to do before I do that is (1) see about creating a brand new branch with a cleaner commit history that's also up to date with master having learned a bit more about git since the last attempt (2) test it and (3) see if I can improve it based on my increased knowledge of Python since its composition.

This should get the bulk of alternate vol/page numbers to show up. There were also a small number of data discrepancies that we discussed in #1523 (ie with Cp texts) but I'm inclined to suggest that those be resolved with #1504.

This is related to but unimproved by #1353 which I'll submit a resolution to as well.

**Comment by sujato** [comment]
See previous comments.

Bv vol/page is not showing up at all on staging. (Staging on left, production on right)

![Screenshot from 2019-11-14 09-50-32](https://user-images.githubusercontent.com/6112010/68867751-581eed80-06c4-11ea-9862-31748c94d723.png)

Also the issue noted by Chan above is not fixed:

> The nerdy row element in parallel items doesn't appear to be working:

![Screenshot from 2019-11-14 09-54-55](https://user-images.githubusercontent.com/6112010/68868136-f8751200-06c4-11ea-9374-92b9d7566e59.png)

**Comment by chansikpark** [comment]
> See previous comments.
> 
> Bv vol/page is not showing up at all on staging. (Staging on left, production on right)

> Can I ask you to specify the steps needed and then we can ensure it is completed.

I seem to have neglected mention here that PR #1537 would require release to staging.

I'm not familiar with the procedures for staging, but I'd gather that if integration was somehow attempted with the PR I submitted and the result was that the Bv vol/page [still](https://github.com/suttacentral/suttacentral/issues/1518) did not show up, it would be because the Arango data loading (via `make load-data`) wasn't performed anew.

**Comment by sujato** [comment]
Normally anything that is merged to Master should be automatically deployed to staging. This was broken until recently but seems to be working now.

**Comment by chansikpark** [comment]
> Also the issue noted by Chan above is not fixed:

Please see [here](https://github.com/suttacentral/suttacentral/pull/1537/commits/69967158a3511dc18ed69c540dbd831a6e3acc2c) for resolution.

**Comment by ihongda** [comment]
Sorry about the slow response! @sujato  @chansikpark 

I will fix this problem as soon as possible.

**Comment by chansikpark** [comment]
Confirmed fixed on staging.

Original case (sutta.json embeds):
![Screenshot from 2019-11-27 13-17-50](https://user-images.githubusercontent.com/16063972/69693648-64219b00-1119-11ea-8ca5-38f67ab4300b.png)

Extended case (html embeds):
![Screenshot from 2019-11-27 13-21-06](https://user-images.githubusercontent.com/16063972/69693687-992ded80-1119-11ea-92ec-a5ab395e0c37.png)

Extended case 2 (remove sutta.json embeds for Dhp for fallback to html):
![Screenshot from 2019-11-27 13-27-50](https://user-images.githubusercontent.com/16063972/69693738-cc707c80-1119-11ea-8436-31fc7b00a42e.png)

Extra case (parallels nerdy row):
![Screenshot from 2019-11-27 13-19-22](https://user-images.githubusercontent.com/16063972/69693661-7ef40f80-1119-11ea-99d8-17365d2202a3.png)
